schema.json

```diff
@@ -87,7 +88,7 @@
   },
   "definitions": {
     "schema": {
-      "$id": "https://json-schema.org/draft-07/schema",
+      "$ref": "https://json-schema.org/draft-07/schema#",
       "type": "object",
       "default": {}
```

Comment:
`https://json-schema.org/draft-07/schema#` has a default value of `true`. This *should* be overriding it, making the autocomplete suggest {}.